task_1/index.html

```diff
@@ -1,47 +1,140 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
 </head>
-
-<body>   
+<body> 
     <div class="wrapper">
-        <div class="header"></div>
+        <div class="header">
+            <div class="coffee-logo">
+                <div><img class="coffee-image-logo" src="./photo/coffee logo.png" alt="coffee logo"></div>
+            </div>
+            <div class="navbar">
+                <div id="home-item" class="header-item active" onclick="onMenuItemClick('home-item')">Home</div>
+                <div id="about-item" class="header-item" onclick="onMenuItemClick('about-item')">About Us</div>
+                <div id="lands-item" class="header-item" onclick="onMenuItemClick('lands-item')">Our Lands</div>
+                <div id="suppliers-item" class="header-item" onclick="onMenuItemClick('suppliers-item')">Our Suppliers</div>
+                <div id="products-item" class="header-item" onclick="onMenuItemClick('products-item')">Products</div>
+                <div id="contact-item" class="header-item" onclick="onMenuItemClick('contact-item')">Contact Us</div>
+            </div>
+            <div class="header-images">
+                <div><img class="shopping-image" src="./photo/shopping cart.png" alt="shopping cart"></div>
+                <div><img class="profile-image" src="./photo/profile image.png" alt="profile image"></div>
+            </div>
+        </div>
         <div class="main">
-            <div class="hero-image"></div>
+            <div class="hero-image">
+                <div class="black-coffee-image"><img src="./photo/black coffee.png" alt="mocha late"></div>
+                <div class="black-coffee-text">
+                    <h4>Mocha Late</h4>
+                    <p class="gray-color">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species.</p>
+                    <div class="distance">
+                        <div>1 Bag</div>
+                        <div>$69.99</div>
+                    </div>
+                </div>
+            </div>
             <div class="hero-banner">
                 <div class="text-banner">
-                    <p>Be Active</p>
-                    <p>Barnsley Brew Coffee</p>
-                    <p>Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species. When coffee berries turn from.</p>
-                    <p>Top Packs</p>
+                    <p class="first-title">Be Active</p>
+                    <h1 class="second-title">Barnsley Brew Coffee</h1>
+                    <p class="second-title-p">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species. When coffee berries turn from.</p>
+                    <p class="third-title">Top Packs</p>
```

Comment:
I think if you use sub-title will be better 
```suggestion
                    <h1 class="second-title">Barnsley Brew Coffee</h1>
                    <p class="second-title-p">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species. When coffee berries turn from.</p>
                    <p class="third-title">Top Packs</p>
```